### PR TITLE
https://issues.redhat.com/browse/ACM-6672 Clarify Subm intro

### PR DESCRIPTION
**Pull request by oafischer**
Also for 2.8 and 2.7

### CODE REVIEW

**Review by nyechiel — APPROVED** [review]

File: networking/submariner/subm_intro.adoc [hunk 1]
```diff
@@ -3,7 +3,7 @@
...
 
-See the link:https://access.redhat.com/articles/7006295[{product-title-short} support matrix] for more details about which infrastructure providers are supported by xref:../submariner/deploy_subm_console.adoc#deploying-submariner-console[automated console deployments] and which infrastructure providers require xref:../submariner/deploy_subm_manual.adoc#deploying-submariner-manually[manual deployment].
+Make sure to see the link:https://access.redhat.com/articles/7006295[{product-title-short} support matrix] for more details about the support levels of infrastructure providers, including which providers support xref:../submariner/deploy_subm_console.adoc#deploying-submariner-console[automated console deployments] or require xref:../submariner/deploy_subm_manual.adoc#deploying-submariner-manually[manual deployment].
```
Comment (sridhargaddam) [inline networking/submariner/subm_intro.adoc]: @nyechiel the link https://access.redhat.com/articles/7006295 points to 2.8 support matrix whereas this PR is targetted for 2.9 
Do we have any support matrix link for 2.9?

Comment (nyechiel) [reply networking/submariner/subm_intro.adoc]: Oh, good catch. I don't think that such a doc exists as of it, but @cadawson or @oafischer should know.

While on this, is there a more elegant way to link the support matrix URL in here (i.e, a shared alias for each branch/release)? It looks like we are manually linking it again and again at the moment.

Comment (cadawson) [reply networking/submariner/subm_intro.adoc]: Here will be the link to the 2.9 support matrix: https://access.redhat.com/articles/7027073. @sridhargaddam and @oafischer .

Comment (cadawson) [inline networking/submariner/subm_intro.adoc]:
```suggestion
Make sure to see the link:https://access.redhat.com/articles/7027073[{product-title-short} support matrix] for more details about the support levels of infrastructure providers, including which providers support xref:../submariner/deploy_subm_console.adoc#deploying-submariner-console[automated console deployments] or require xref:../submariner/deploy_subm_manual.adoc#deploying-submariner-manually[manual deployment].
```

**Review by cadawson — APPROVED** [review]
Looks good. Added the draft 2.9 support matrix link. Thanks!

File: networking/submariner/subm_intro.adoc (hunk 1 above)
Comment (oafischer) [reply networking/submariner/subm_intro.adoc]: Awesome catch, thanks! We don't actually link the matrix that often in the docs, but a conref might still make sense. @cadawson Do you know if links work as conrefs?

**Pull request merged**